### PR TITLE
WIP: add route to get the invoices of the customer

### PR DESCRIPTION
**Pull request by Pilzington**

### CODE REVIEW

**Comment by majst01** [comment]
Pdf is not a string, must be a []byte. Are we really offering PDF Downloads ?

**Comment by Pilzington** [comment]
> Pdf is not a string, must be a []byte. Are we really offering PDF Downloads ?

Stripe provides a link to invoice pdfs, which is hosted there. I want to write the link in this field.

**Comment by majst01** [comment]
> > Pdf is not a string, must be a []byte. Are we really offering PDF Downloads ?
> 
> Stripe provides a link to invoice pdfs, which is hosted there. I want to write the link in this field.

but then naming should be different

File: proto/api/v1/payment.proto [hunk 1]
```diff
@@ -74,6 +77,13 @@ message SubscriptionUsageItem {
   google.protobuf.Timestamp period_end = 5;
 }
 
+message Invoice {
+  string invoice_id = 1;
```
Comment (majst01) [inline proto/api/v1/payment.proto:81]: still Invoice.InvoiceID is weird, rename it simply to `id`

**Review by majst01 — APPROVED** [review]

**Pull request merged**